### DISCUSSION TITLE
PNG Export resolution

### DISCUSSION THREAD

**Discussion by cavaterra-a** [SSShooter/mind-elixir-core#286] [Q&A]
Hello, is there any way I can tweak and control the `exportPng` resolution size? I tried fiddling around with `nodes, lines, container` with no luck, any hints? Thanks in advance

**Comment by SSShooter** [comment]
Hi, I think you can write your own export function base on the original one.
 https://github.com/SSShooter/mind-elixir-core/blob/3c8bd4425ba025a893fa2846737f92978e6edd11/src/plugin/exportImage.ts#L245

**Comment by cavaterra-a** [comment]
Yep that's what i ended up doing, I thought there could be an easier way with `injectCss`, thanks!
